imgkap 1.13 (new formula)

nohal opened the pull request:
### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [ ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Have you built your formula locally prior to submission with `brew install <formula>`?


This code is considered mature and stable for several years, given the target audience it is pretty unlikely it ever attracts enough Github developer attention to pass "GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)".

DomT4 commented:
> This code is considered mature and stable for several years

Was it hosted anywhere else before this?

nohal commented:
Yes, at http://www.dacust.com/inlandwaters/imgkap/

dunn commented on Library/Formula/imgkap.rb:
```diff
@@ -0,0 +1,17 @@
+  end
+
+  test do
+    system "imgkap || true"
```
You can test exit code with e.g. `shell_output("command", 1)`, but does this test check the software functionality?

nohal commented:
I have updated upstream with sample data and changed the test to perform a full conversion operation, there is IMO not much more that can be done to check whether the tool is working.

mikemcquaid commented:
> This code is considered mature and stable for several years, given the target audience it is pretty unlikely it ever attracts enough Github developer attention to pass "GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)".

My concern is that this is effectively a fork of the original project: "This repository is a Github clone of the original code created by M'dJ in 2011 and maintained by Pavel Kalian" and it doesn't have a lot of GitHub interest. That's not to say it's not valuable software but it's maybe worth considering [creating your own tap](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md) instead. Thanks!

nohal commented:
The original author is not available anymore, that's why I took over the maintainership. Yes, it has and will have very little Github interest given the target audience is very little techie - that's why I tried to make it as easy as possible for them to get it installed. If you feel like it makes no sense for HB to have such a formula, just close the PR and I will find other way to distribute it, being it a tap or otherwise.

mikemcquaid commented:
I'll let some other maintainers give their views.

apjanke commented:
Yeah. As a fork or a takeover of abandonware, I think we'd want to let this mature a while before inclusion in the core Homebrew repo, especially since there's no package name change to make the change in ownership apparent. Provenance matters.

Providing it in your own tap like @mikemcquaid suggests sounds like the way to go for now. Wouldn't be much harder for users to install, in technical terms.

I and probably other maintainers would still be happy to do code review etc on your formula. You can submit them as PRs to your own tap and send an email with a link and request for review to the  [homebrew-discuss mailing list](https://groups.google.com/forum/#!forum/homebrew-discuss) to get feedback even if it's not going to be included in the core repo.

nohal commented:
OK, going to take that route.
Thanks for the feedback
